src/components/tree/phyloTree/change.js

```diff
@@ -306,9 +306,6 @@ export const change = function change({
     elemsToUpdate.add(".grid").add(".regression");
     svgPropsToUpdate.add("cx").add("cy").add("d").add("opacity").add("visibility");
   }
-  if (newLayout) {
-    useModifySVGInStages = true;
-  }
 
```

Comment:
:100: This type of refactor is small but I think really helpful to iteratively making complex logic more understandable/approachable.